forest_example.ipynb

```diff
@@ -354,7 +354,7 @@
    "outputs": [],
    "source": [
     "loaded_preds = loaded_clf.predict_proba(X_test)\n",
-    "loaded_y_pred = np.vectorize(preds_mapper.get)(np.argmax(loaded_preds, axis=1))\n",
+    "loaded_y_pred = np.vectorize(preds_mapper.get)(np.argmax(loaded_preds, axis=1)).astype(int)\n",
```

Comment:
I think the astype here is useless now no? I think there are no changes at all to make on this notebook